integration-tests/package.json

```diff
@@ -13,7 +13,7 @@
     "author": "",
     "license": "Apache-2.0",
     "dependencies": {
-        "@frequency-chain/api-augment": "v0.9.29",
+        "@frequency-chain/api-augment": "v0.9.30",
```

Comment:
how is this going to affect metadata for other libraries?

Reply:
I think exacting the version might complicate this we should do "^v0.9.29"for now given I am not sure at what stage these tests are run but for sure npm wont have this version till we release